assets/js/components/dialog.js

```diff
@@ -66,7 +67,10 @@ class Dialog extends Component {
 		return (
 			<div
 				ref={ this.dialogRef }
-				className={ `mdc-dialog ${ dialogActive ? 'mdc-dialog--open' : '' }` }
+				className={ classnames(
+					'mdc-dialog',
+					{ 'mdc-dialog--open': dialogActive } )
```

Comment:
I can't find the original comment but saw a PR email about this one…

It feels like something ESLint _should_ complain about and certainly can, but I'm guessing we don't have that sort of rule enabled.

I'd happily add it, but I don't feel it's worth blocking the PR on 😄